public.php

```diff
@@ -55,7 +60,24 @@
 		$pathInfo = trim($pathInfo, '/');
 		list($service) = explode('/', $pathInfo);
 	}
-	$file = \OC::$server->getConfig()->getAppValue('core', 'public_' . strip_tags($service));
+
+	// Managing some .well-known request
+	OC_App::loadApps();
```

Comment:
Should we keep the current load calls in order?
```suggestion
	OC_App::loadApps(['authentication']);
	OC_App::loadApps(['filesystem', 'logging']);
	OC_App::loadApps();
```